### PR TITLE
removing total lines of code updates from back end, not including table storage

### PR DESCRIPTION
**Pull request by hannahbergam**
This PR continues the goal of removing total lines of code, without disrupting the messages at the end of levels that state "You just wrote [x] lines of code!". 

This PR does not go through the trouble of removing total_lines from the User table, which may be another follow up task. For now, I'm hoping to remove writes to this table so that we are no longer updating the metric. 

All total lines of code displays have already been removed from the front end, so this is just the back-end work to match (relevant PR linked below).

## Links

A follow up from this pr: https://github.com/code-dot-org/code-dot-org/pull/46934

- spec: []()
- jira ticket: []https://codedotorg.atlassian.net/browse/LP-2291

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/clientState.js [hunk 1]
```diff
@@ -159,33 +140,6 @@ function levelProgressByScript() {
...
-};
-
-/**
- * Adds the given number of completed lines.
- * @param {number} addedLines
- */
-function addLines(addedLines) {
-  var newLines = Math.min(
-    clientState.lines() + Math.max(addedLines, 0),
-    MAX_LINES_TO_SAVE
```
Comment (bethanyaconnor) [inline apps/src/code-studio/clientState.js:183]: I'm curious why this logic was here? Why couldn't we add 2000 lines to the total? (You might not know the answer but I'm very curious...)

**Review by bethanyaconnor — COMMENTED** [review]
Did we find a place in one of the controllers where we were updating lines of code? Should we remove that logic too?

File: apps/src/code-studio/clientState.js (hunk 1 above)
Comment (hannahbergam) [reply apps/src/code-studio/clientState.js:183]: I found a very very old PR about this.... https://github.com/code-dot-org/code-dot-org/pull/641

**Comment by hannahbergam** [comment]
> Did we find a place in one of the controllers where we were updating lines of code? Should we remove that logic too?

Gah yes- thanks for the reminder! I found it and pushed a new commit. Additionally... there are a few controller tests that I believe relate to what the table should allow for that column (example: doesn't allow negative numbers for total_lines). Should I remove all these tests now or leave that to be part of a part 3 where we actually remove the column from the table?

File: apps/src/code-studio/clientState.js (hunk 1 above)
Comment (hannahbergam) [reply apps/src/code-studio/clientState.js:183]: lolol and the title of the PR pretty much says it all "don't allow people to send us nonsense numbers to mess up the lines of code counter"

**Comment by bethanyaconnor** [comment]
> Additionally... there are a few controller tests that I believe relate to what the table should allow for that column (example: doesn't allow negative numbers for total_lines). Should I remove all these tests now or leave that to be part of a part 3 where we actually remove the column from the table?

If they are in `activities_controller_test.rb` then yes as the logic there is gone. If there is other logic (maybe in `user_test.rb` or something) then maybe not

**Comment by hannahbergam** [comment]
> > Additionally... there are a few controller tests that I believe relate to what the table should allow for that column (example: doesn't allow negative numbers for total_lines). Should I remove all these tests now or leave that to be part of a part 3 where we actually remove the column from the table?
> 
> If they are in `activities_controller_test.rb` then yes as the logic there is gone. If there is other logic (maybe in `user_test.rb` or something) then maybe not

okay I *think* I've found and removed all relevant controller tests but would love you to look over the new stuff I committed!

**Review by bethanyaconnor — APPROVED** [review]
🎉 assuming Drone passes looks great!

**Pull request merged**